index.html

```diff
@@ -1,530 +1,2 @@
-<!DOCTYPE HTML>
-<!--
-    Prologue by HTML5 UP
-    html5up.net | @n33co
-    Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
--->
-<html>
-    <head>
-	<title>Julia Astro</title>
-	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
-	<meta name="description" content="" />
-	<meta name="keywords" content="" />
-	<link rel="shortcut icon" href="images/logo_32.png">
-	<!--[if lte IE 8]><script src="css/ie/html5shiv.js"></script><![endif]-->
-	<script src="js/jquery.min.js"></script>
-	<script src="js/jquery.scrolly.min.js"></script>
-	<script src="js/jquery.scrollzer.min.js"></script>
-	<script src="js/skel.min.js"></script>
-	<script src="js/skel-layers.min.js"></script>
-	<script src="js/init.js"></script>
-	<noscript>
-	    <link rel="stylesheet" href="css/skel.css" />
-	    <link rel="stylesheet" href="css/style.css" />
-	    <link rel="stylesheet" href="css/style-wide.css" />
-	    <link rel="stylesheet" href="css/custom.css" />
-	</noscript>
-	<!--[if lte IE 9]><link rel="stylesheet" href="css/ie/v9.css" /><![endif]-->
-	<!--[if lte IE 8]><link rel="stylesheet" href="css/ie/v8.css" /><![endif]-->
-    </head>
-    <body>
+<meta http-equiv="refresh" content="0; url=http://juliaphysics.github.io/latest/index.html" />

```

Comment:
I assume you wanted to redirect to JuliAstro website :wink: 
```suggestion
<meta http-equiv="refresh" content="0; url=https://juliaastro.github.io/dev/index.html" />
```